src/menu/index.unit.tsx

```diff
@@ -50,7 +50,7 @@ describe('Menu', () => {
   it('Should configure onClick listeners on nested menu items', () => {
     // Set up the second menu item with a mocked onClick handler.
     const onClickMock = jest.fn()
-    const props = {...defaultProps}
+    const props = JSON.parse(JSON.stringify(defaultProps))
```

Comment:
??

Reply:
What do you mean?
The spread does not deep clone an object, we need that since we mutate the default props below. Any suggestions?

Reply:
would it work adding this `const defaultProps` in a beforeEach to reset it on each test?

```
describe('Menu', () => {
  let defaultProps = {}
  beforeEach(() => {
    defaultProps = {
      items: [
          {
          id: 'menu-item-1',
          label: 'Dashboard',
          leftAddon: <HomeIcon />,
          href: '/',
          },
          {
          id: 'menu-item-2',
          label: 'Rides offered',
          leftAddon: <NewspaperIcon />,
          rightAddon: <CheckShieldIcon />,
          href: '/rides',
          },
      ],
    }
  })

  it('...', () => {}) 
})
```

Reply:
I like it, done! Thanks guys